virtio-devices/src/vhost_user/net.rs

```diff
@@ -127,9 +129,36 @@ impl Net {
         vu_cfg: VhostUserConfig,
         server: bool,
         seccomp_action: SeccompAction,
+        restoring: bool,
     ) -> Result<Net> {
         let mut num_queues = vu_cfg.num_queues;
 
+        if restoring {
+            // We need 'queue_sizes' to be correct since VirtioPciDevice::new()
+            // will create the queues based on this information.
```

Comment:
What does this comment mean? Do you mean you need to already have included the control queue?

Reply:
Yes because if you don't, then you'll only get 2 queues from `VirtioPciDevice`. I've checked the `activate()` common code, which only checks for the number of `queue_evts` being identical to the number of `queues`. I don't think it's harmful to be in a case where we would "create" more queues than used.

Reply:
My point is about the comment. I don't think we'll remember what "needs to be correct" means in a years time. Perhaps you could simply explain that by including the control queue there is guaranteed to be sufficient queues allocated. Also if num_queues > 2 then you MUST have a MQ control queue (by the spec.)

Reply:
Ah gotcha, I'll update the comment :)

Reply:
Done